src/dbcp/metadata/private_data_warehouse.py

```diff
@@ -1,14 +1,93 @@
 """SQL Alchemy metadata for the datawarehouse tables."""
 
-from sqlalchemy import Column, MetaData, String, Table
+from sqlalchemy import (
+    BigInteger,
+    Boolean,
+    Column,
+    Float,
+    Integer,
+    MetaData,
+    String,
+    Table,
+)
 
 metadata = MetaData()
 schema = "private_data_warehouse"
 
-private_table = Table(
-    "private_table",
+acp_projects = Table(
+    "acp_projects",
     metadata,
-    Column("id", String, primary_key=True),
-    Column("private_data", String, primary_key=True),
+    Column("proj_id", BigInteger, primary_key=True),
+    Column("status", String),
+    Column("eia_plant_code", Integer),
+    Column("project_name", String),
+    Column("phase_name", String),
+    Column("phase_type", String),
+    Column("developers", String),
+    Column("owners", String),
+    Column("iso_rtos", String),
```

Comment:
Do the values of this field align with lbnl/gridstatus `iso_region` fields? If so maybe we use the same field name?

Reply:
ah ya good catch. I'll standardize some other column names too, like `plant_id_eia` and `capacity_mw`